net.lutris.Lutris.yml

```diff
@@ -24,7 +24,6 @@ finish-args:
   # For Debian
   - --filesystem=/media
   - --filesystem=~/.var/app/com.valvesoftware.Steam:ro
-  - --filesystem=xdg-config/MangoHud:ro
```

Comment:
@strycore You don't need to remove it. Just change `xdg-config` to `~/.config`.

See [this change](https://github.com/flathub/org.gnome.Evolution/commit/739668db1167cbe41706ed21b8633b5fe122d402#diff-f826885406e129b570972835169d35cff130e7cd061ec5b25c3fea8dd878edcbL32) in GNOME Evolution as an example.